services/parser-database/apiSpec.yaml

```diff
@@ -0,0 +1,130 @@
+swagger: "2.0"
+info:
+  description: "API description for the Parser/Database component of the Road Search Engine."
+  version: "1.0.0"
+  title: "Parser/Database Component"
+  contact:
+    email: "jaimehisao@google.com"
+host: "localhost"
+basePath: "/v1"
+schemes:
+- "https"
+- "http"
+paths:
+  /parse:
+    post:
+      tags:
+      - "parser"
+      summary: "Trigger a document download, parse, and metadata tagging"
+      responses:
+        "200":
+         description: "Document was parsed successfully"
+        "409":
+          description: "Parser service is already running."
+        "500":
+          description: "Server cannot parse the document and add it to the database. Likely if the document is not in the specified URL or if the database is offline."
+  /articles/byKeyword:
+    post:
+      tags:
+      - "articles"
+      summary: "Finds any article that contains the specified keyword(s)"
+      description: ""
+      operationId: "returnArticleIDs"
+      consumes:
+      - "application/json"
+      produces:
+      - "application/json"
+      parameters:
+      - in: "body"
+        name: "body"
+        description: "Keywords that need to be matched with an article in the database"
+        required: true
+        schema:
+          $ref: '#/definitions/ArticleKeywordsQuery'
+      responses:
+        "400":
+          description: "No articles matched keyword query"
```

Comment:
nit: Does it make sense to return an error code instead of just an empty list? Usually, error codes indicate that something was *wrong* with the request.